src/general/OptionsMenu.cs

```diff
@@ -501,6 +506,12 @@ public override void _Ready()
     {
         ResolveNodeReferences(true);
 
+        foreach (var mode in typeof(Settings.StrainBarVisibility).GetEnumValues().Cast<Settings.StrainBarVisibility>())
```

Comment:
For the options menu the usual way to setup these dropdowns is to set them in the Godot editor and not load them manually like this.